bundles/org.openhab.binding.ventaair/src/main/java/org/openhab/binding/ventaair/internal/VentaThingHandler.java

```diff
@@ -0,0 +1,338 @@
+                    int timer = ((DecimalType) command).intValue();
+                    dispatchActionToDevice(new TimerAction(timer));
+                }
+                break;
+            case VentaAirBindingConstants.CHANNEL_SLEEP_MODE:
+                if (command instanceof OnOffType) {
+                    OnOffType sleepMode = (OnOffType) command;
+                    dispatchActionToDevice(new SleepModeAction(sleepMode == OnOffType.ON ? true : false));
+                }
+                break;
+            case VentaAirBindingConstants.CHANNEL_BOOST:
+                if (command instanceof OnOffType) {
+                    OnOffType boost = (OnOffType) command;
+                    dispatchActionToDevice(new BoostAction(boost == OnOffType.ON ? true : false));
+                }
+                break;
+            case VentaAirBindingConstants.CHANNEL_CHILD_LOCK:
+                if (command instanceof OnOffType) {
+                    OnOffType childLock = (OnOffType) command;
+                    dispatchActionToDevice(new ChildLockAction(childLock == OnOffType.ON ? true : false));
+                }
+                break;
+            case VentaAirBindingConstants.CHANNEL_AUTOMATIC:
+                if (command instanceof OnOffType) {
+                    OnOffType automatic = (OnOffType) command;
+                    dispatchActionToDevice(new AutomaticAction(automatic == OnOffType.ON ? true : false));
+                }
+                break;
+
+            default:
+                break;
+        }
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(VentaAirDeviceConfiguration.class);
+
+        updateStatus(ThingStatus.UNKNOWN);
+
+        String configErrorMessage;
+        if ((configErrorMessage = validateConfig()) != null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, configErrorMessage);
+            return;
+        }
+
+        // we all know that 42 is the answer to everything, so let's pick this one ;)
+        BigDecimal hash = config.hash != null ? config.hash : new BigDecimal("-42");
+
+        Header header = new Header(config.macAddress, config.deviceType.intValue(), hash.toString(), "openHAB");
+
+        communicator = new Communicator(config.ipAddress, header, config.pollingTime, new StateUpdatedCallback());
+        communicator.startPollDataFromDevice(scheduler);
+    }
+
+    private @Nullable String validateConfig() {
+        if (config.ipAddress == null || config.ipAddress.isEmpty()) {
+            return "IP address not set";
+        }
+        if (config.macAddress == null || config.macAddress.isEmpty()) {
+            return "Mac Address not set, use discovery to find the correct one";
+        }
+        if (config.deviceType == null) {
+            return "Device Type not set, use discovery to find the correct one";
+        }
+        if (config.pollingTime == null || config.pollingTime.compareTo(BigDecimal.ZERO) < 0) {
```

Comment:
```suggestion
        if (config.pollingTime == null || config.pollingTime.compareTo(BigDecimal.ZERO) <= 0) {
```
If it must be more than zero, consider what happens if it is exactly 0.